src/dbus_mpris.rs

```diff
@@ -438,42 +439,54 @@ async fn create_dbus_server(
         let mut seeked_position = None;
 
         // Update playback state from event
-        let (track_id, playback_status) = match event {
+        let (track_id, playback_status, player_volume) = match event {
+            PlayerEvent::VolumeSet { volume } => {
+                (last_track_id, last_playback_status, Some(volume))
+            }
             PlayerEvent::Playing { track_id, position_ms, .. } => {
                 seeked_position = Some(position_ms);
-                (Some(track_id), Some(PlaybackStatus::Playing))
+                (Some(track_id), Some(PlaybackStatus::Playing), last_volume)
             }
             PlayerEvent::Stopped { .. } => {
-                (last_track_id, Some(PlaybackStatus::Stopped))
+                (last_track_id, Some(PlaybackStatus::Stopped), last_volume)
             }
             PlayerEvent::Paused { .. } => {
-                (last_track_id, Some(PlaybackStatus::Paused))
+                (last_track_id, Some(PlaybackStatus::Paused), last_volume)
             }
-            _ => (last_track_id, last_playback_status),
+            _ => (last_track_id, last_playback_status, last_volume),
```

Comment:
We should be able to use `continue` here:

```suggestion
            _ => continue,
```

which might increase the readability. This is, of course, not an issue with your changes, just came to my mind while rereading the code.

Reply:
That part was unchanged (apart from the addition of last_volume).. 
Let's see what @MDeiml thinks.

Reply:
continue should be fine there, yes